src/EnergyPlus/WindowManager.cc

```diff
@@ -689,6 +707,9 @@ namespace WindowManager {
 
 			lquasi = false;
 			AllGlassIsSpectralAverage = true;
+			int TotalIPhi = 10;
```

Comment:
The value of 10 is OK as an initial value. This is true for Data Type = Spectral. If SpectralAndAngle, the value will be changed.